src/SFML/Graphics/Shader.cpp

```diff
@@ -959,6 +1036,74 @@ int Shader::getUniformLocation(const std::string& name)
 
 namespace sf
 {
+////////////////////////////////////////////////////////////
+Shader::Shader(const std::filesystem::path& /* filename */, Type /* type */)
+{
+    throw std::runtime_error("Failed to load shader");
```

Comment:
Do we want to provide more information like `Shader is not supported with OpenGL ES 1`?